service/src/main/java/bio/terra/landingzone/library/configuration/LandingZoneTestingConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.landingzone.library.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "landingzone.testing")
+public class LandingZoneTestingConfiguration {
+
+  private boolean allowAttach;
```

Comment:
Assume this config will be set to true for BEE-only in terra-helmfile (somewhere around [here](https://github.com/broadinstitute/terra-helmfile/blob/master/values/app/workspacemanager/bee.yaml.gotmpl))?

Does it need to be defined in application.yml (which I think exists in [WSM](https://github.com/DataBiosphere/terra-workspace-manager/blob/main/service/src/main/resources/application.yml#L158)) or does it default to `false`?

Reply:
Yeah, this'll be a helmfile change; it defaults to false.